gokart/task.py

```diff
@@ -52,6 +76,10 @@ class TaskOnKart(luigi.Task):
     redis_host = luigi.Parameter(default=None, description='Task lock check is deactivated, when None.', significant=False)
     redis_port = luigi.Parameter(default=None, description='Task lock check is deactivated, when None.', significant=False)
     redis_timeout = luigi.IntParameter(default=180, description='Redis lock will be released after `redis_timeout` seconds', significant=False)
+    redis_fail: bool = gokart.parameter.ExplicitBoolParameter(
```

Comment:
Is there any reason not to use luigi.BoolParameter?

Reply:
No, I think luigi.BoolParameter is okay, so I'll change.